extensions/Worker.Extensions.Storage.Blobs/src/BlobInputAttribute.cs

```diff
@@ -1,11 +1,12 @@
 ﻿// Copyright (c) .NET Foundation. All rights reserved.
 // Licensed under the MIT License. See License.txt in the project root for license information.
 
+using Microsoft.Azure.Functions.Worker.Converters;
 using Microsoft.Azure.Functions.Worker.Extensions.Abstractions;
 
 namespace Microsoft.Azure.Functions.Worker
 {
-    [SupportsDeferredBinding]
+    [InputConverter(disableConverterFallback: false, typeof(BlobStorageConverter))]
```

Comment:
I think I'd like to to find a better name for this (disableConverterFallback ) parameter. The purpose of it is to enable or disable using the builtin converters right? So maybe `UseBuiltinConverters`? or something along those lines?

I think in terms of ease of usability, can we default it to false? It would be nice to also not require a bool value; so for example:

`[InputConverter(typeof(BlobStorageConverter))]`

^ does not use builtin converters

`[InputConverter(UseBuiltinConverters, typeof(BlobStorageConverter))]`

^ would use builtin converters

Reply:
I am with you on the naming issue. I think the intent is that by default all converters will support fallback to default converters. But we will give option to the converter to disable fallback using this flag. We can use name `DisableFallbackToBuiltinConverters` but I am not sure if we want to refer them as `default` or `builtIn` converters? We certainly need to brainstorm on naming at various places. Will wait for the design to get finalized and come back to it in case there is any significant change in flow.